src/flowops_library.c

```diff
@@ -22,7 +22,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#include "config.h"
+#include "../config.h"
```

Comment:
I do not think this is required.. We use -I. -I.. while building so the header is picked up. Can you pass the same -I flags to the android compiler?